pkg/server/sandbox_portforward_windows.go

```diff
@@ -19,19 +19,66 @@ limitations under the License.
 package server
 
 import (
+	"bytes"
+	"fmt"
 	"io"
 
-	"github.com/containerd/containerd/errdefs"
+	"github.com/pkg/errors"
 	"golang.org/x/net/context"
-	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"k8s.io/kubernetes/pkg/kubelet/util/ioutils"
+	"k8s.io/utils/exec"
+
+	ctrdutil "github.com/containerd/cri/pkg/containerd/util"
+	sandboxstore "github.com/containerd/cri/pkg/store/sandbox"
 )
 
-// PortForward prepares a streaming endpoint to forward ports from a PodSandbox, and returns the address.
-// TODO(windows): Implement this for windows.
-func (c *criService) PortForward(ctx context.Context, r *runtime.PortForwardRequest) (*runtime.PortForwardResponse, error) {
-	return nil, errdefs.ErrNotImplemented
+func (c *criService) portForward(ctx context.Context, id string, port int32, stream io.ReadWriter) error {
+	stdout := ioutils.WriteCloserWrapper(stream)
+	stderrBuffer := new(bytes.Buffer)
+	stderr := ioutils.WriteCloserWrapper(stderrBuffer)
+	cmd := []string{"wincat.exe", "localhost", fmt.Sprint(port)}
+	err := c.execInSandbox(id, cmd, stream, stdout, stderr)
+	if err != nil {
+		return errors.Wrapf(err, "failed to execute port forward in sandbox: %s", stderrBuffer.String())
+	}
+
+	return nil
 }
 
-func (c *criService) portForward(ctx context.Context, id string, port int32, stream io.ReadWriter) error {
-	return errdefs.ErrNotImplemented
+func (c *criService) execInSandbox(sandboxID string, cmd []string, stdin io.Reader, stdout, stderr io.WriteCloser) error {
+	ctx := ctrdutil.NamespacedContext()
+	opts := execOptions{
```

Comment:
nit: Prefer moving the variable definition to where it will be used.

Reply:
Done.